pkg/handler/handler.go

```diff
@@ -34,6 +36,31 @@ import (
 	"k8s.io/kubernetes/pkg/apis/core/v1"
 )
 
+type podUpdateSettings struct {
+	skipContainers map[string]bool
+	useRegionalSTS bool
+}
+
+// newPodUpdateSettings returns the update settings for a particular pod
+func newPodUpdateSettings(annotationDomain string, pod *corev1.Pod, useRegionalSTS bool) *podUpdateSettings {
+	settings := &podUpdateSettings{
+		useRegionalSTS: useRegionalSTS,
+	}
+
+	skippedNames := map[string]bool{}
+	skipContainersKey := annotationDomain + "/" + pkg.SkipContainersAnnotation
+	if value, ok := pod.Annotations[skipContainersKey]; ok {
+		r := csv.NewReader(strings.NewReader(value))
+		// error means we don't skip any
```

Comment:
while it's true that running into an error parsing the annotation here means we won't skip any containers, that parse error is essentially buried with no indication to the user that they supplied a bad annotation value :)

Perhaps consider logging a message about malformed annotation value here?

Reply:
Added logging